lib/internal/cluster/utils.js

```diff
@@ -4,10 +4,10 @@ const util = require('util');
 module.exports = {
   sendHelper,
   internal,
-  handles: {} // Used in tests.
+  handles: new Map() // Used in tests.
```

Comment:
If this is used in tests, shouldn't tests be updated?

Reply:
@targos the only test that was using it was removed. See https://github.com/nodejs/node/pull/23131